docs/src/components/CombinationNew.js

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import { Flex } from 'gestalt';
+import React, { type Node } from 'react';
+import MainSectionCard from './MainSectionCard.js';
+
+type Props = {
```

Comment:
Can we close this type?

Reply:
Given how this component works, I don't actually think we can. The extra props added get used to render the combinations